build-scripts/gh-actions-setup-inv

```diff
@@ -6,6 +6,8 @@ EOF
 "
 sudo chmod 755 /usr/sbin/policy-rc.d
 sudo apt-get update
+# Avoid GRUB related errors due to runner image configuration by reinstalling it.
```

Comment:
Please add FIXME: because I'd love for us to remove this again when we can. Thanks!

Reply:
Sure!